plugins/in_tail/tail_fs_inotify.c

```diff
@@ -51,7 +51,7 @@ static int debug_event_mask(struct flb_tail_config *ctx,
     }
 
     if (file) {
-        buf_size = file->name_len + 64;
+        buf_size = file->name_len + 128;
```

Comment:
128 covers:

an inode ID is a 32 bit integer, so 10 chars max, let's say 12

18 chars in the first format
10 - 14 chars per event, 5 events possible
= 100 chars could be added, I will make it 128 for a nice power of 2